Dockerfile-csi-host-definer

```diff
@@ -0,0 +1,32 @@
+FROM registry.access.redhat.com/ubi8/python-38:1-75.1638364053 as builder
+
+RUN pip3 install --ignore-installed --default-timeout=100 --upgrade pip==21.2.4
+WORKDIR /tmp
+COPY host_definer/requirements.txt .
+RUN pip3 install -r ./requirements.txt
+
+FROM registry.access.redhat.com/ubi8/python-38:1-96
+
+ARG VERSION=1.10.0
+ARG BUILD_NUMBER=0
+
+###Required Labels
+LABEL name="IBM block storage CSI driver hostDefiner" \
+      vendor="IBM" \
+      maintainer="IBM Storage" \
+      version=$VERSION \
+      release=$BUILD_NUMBER \
+      summary="The hostDefiner component of the IBM block storage CSI driver" \
```

Comment:
technically it's not really part of the CSI driver, because it's not really part of the CSI spec

Reply:
so you think it better to represent it like this: "The host definer component of the IBM block storage"? or just write about it like: "The host definer defines Kubernetes nodes on IBM  block storage"?

Reply:
the latter

Reply:
done